src/Altinn.Platform/Altinn.Platform.Events/Events/Migration/v0.03/01-setup-tables.sql

```diff
@@ -0,0 +1,41 @@
+-- Table: events.eventssubscription
+
+CREATE TABLE IF NOT EXISTS events.eventssubscription
+(
+    id BIGSERIAL,
+    sourcefilter character varying COLLATE pg_catalog."default",
+    subjectfilter character varying COLLATE pg_catalog."default",
+    typefilter character varying COLLATE pg_catalog."default",
+    consumer character varying COLLATE pg_catalog."default" NOT NULL,
+    endpointurl character varying COLLATE pg_catalog."default" NOT NULL,
+    createdby character varying COLLATE pg_catalog."default" NOT NULL,
+    validated BOOLEAN NOT NULL,
+    "time" timestamptz  NOT NULL,
+    CONSTRAINT eventssubscription_pkey PRIMARY KEY (id)
+)
+
+
+TABLESPACE pg_default;
+
```

Comment:
Missing grant of privileges to platform_events. 
This must be done for every new table if the default privileges for the user is not altered.

Reply:
Added grants file from 01